### PR TITLE
Add list tweets endpoint

### PR DESCRIPTION
**Pull request by blizniukov**

### CODE REVIEW

**Comment by redouane59** [comment]
Can you remove the indentation changes please ? Or use the project code style ?

**Comment by redouane59** [comment]
It's a duplicate of `getTweets` method no ?

**Comment by redouane59** [comment]
Don't you think it's better to call the method as the other one with only one argument `getList` ?

**Comment by redouane59** [comment]
Let me one second to review it deeper, maybe I was wrong and the endpoint you call is not exacting the same one.

File: src/main/java/io/github/redouane59/twitter/ITwitterClientV2.java [hunk 1]
```diff
@@ -594,6 +594,14 @@ public interface ITwitterClientV2 {
    */
   TwitterList getList(String listId);
 
+  /**
+   * Get a tweet list by list id calling https://api.twitter.com/2/tweets
```
Comment (redouane59) [inline src/main/java/io/github/redouane59/twitter/ITwitterClientV2.java:598]: are you sure about it ?
Isn't it https://api.twitter.com/2/lists/:id/tweets instead ?

Comment (blizniukov) [reply src/main/java/io/github/redouane59/twitter/ITwitterClientV2.java:598]: My bad. Yes it is lists/tweets one. I will fix doc

Comment (redouane59) [reply src/main/java/io/github/redouane59/twitter/ITwitterClientV2.java]: Ok now I can understand better what you wanted to implement and it makes sense to have another method name :)

**Pull request merged**